samples/matter/template/sample.yaml

```diff
@@ -24,13 +18,9 @@ tests:
       - nrf21540dk_nrf52840
     platform_allow: nrf52840dk_nrf52840 nrf21540dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
   # ---------------
-  sample.matter.template.debug.nrf5340dk:
+  sample.matter.template.debug:
```

Comment:
Why don't we have nrf52840dk_nrf52840 in the intergration_platforms section here and in every other sample?

Reply:
Integration with nrf52840dk_nrf52840 is covered by sample.matter.lock.smp_dfu_ffs it is also taken for system tests later, maybe we need to include more configurations with nrf52840dk_nrf52840 but better to start discussion on this with this document  https://projecttools.nordicsemi.no/confluence/display/KRKNWK/Configurations+Selection+for+Integration+Builds